docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -116,6 +116,25 @@ Custom yaml settings are silently ignored by {fleet}.
 Settings with input blocks, such as Max agents are still effective.
 ====
 
+[[known-issue-issue-2249-8.6.2]]
+.{agent} does not load fleet configuration after upgrade from unencrypted config
+[%collapsible]
+====
+*Details* +
+{agent} will not load `fleet.yml` information after upgrading to a version that uses encrypted configuration (> 8.3).
+For a managed agent the symptom manifests as the inability to connect back to fleet after upgrade and the following log:
+
+[source,shell]
+----
+Error: fleet configuration is invalid: empty access token
+...
+----
+For more information refer to {agent-issue}2249[#2249].
+
+*Impact* +
+{agent} loses the ability to connect back to {fleet} after upgrade.
+Fix for this issue is available in version 8.7.0 and higher, for versions 8.3-8.6 a re-enroll is necessary for the agent to connect back to {fleet}
```

Comment:
Does this actually affect 8.3-8.6? I thought it only affected 8.6?

Reply:
I thought I had tested 8.3 and 8.4 in the original issue but I actually didn't, I only tested 8.2 -> 8.6
I will test 8.2 -> 8.3, 8.2 -> 8.4, 8.2 ->8.5 upgrade scenario to confirm

Reply:
8.2.0 -> 8.3.3, 8.2.0 ->8.4.3, 8.2.0 -> 8.5.3 all work correctly I will update the description to reflect that